Mathlib/Tactic/Push.lean

```diff
@@ -229,9 +210,6 @@ distrib mode it produces `¬p ∨ ¬q`. To use distrib mode, use `set_option pus
 -/
 elab "push_neg" loc:(location)? : tactic =>
   let loc := (loc.map expandLocation).getD (.targets #[] true)
-  withLocation loc
-    pushNegLocalDecl
-    pushNegTarget
-    (fun _ ↦ logInfo "push_neg couldn't find a negation to push")
+  atLocation pushNegCore "push_neg" (failIfUnchanged := true) false loc
```

Comment:
This changes the error message. (Do we have a test for this? Can you add one?) I think the deduplication is nicer than trying to preserve the exact wording.